nautobot/project-static/src/scss/nautobot.scss

```diff
@@ -477,7 +478,15 @@ $s2bs5-border-color: $border-color;
 
     /* Focus styles */
     --#{$prefix}focus-ring-color: #{$focus-ring-color-dark};
+
+    @include meta.load-css('flatpickr/dist/themes/dark.css');
+  }
+
+  @include color-mode(light, true) {
+    @include meta.load-css('flatpickr/dist/themes/light.css');
   }
+} @else {
+  @include meta.load-css('flatpickr/dist/themes/light.css');
```

Comment:
What's the case where this else is needed? Won't we always have `$enable-dark-mode`?